crates/bindings/src/lib.rs

```diff
@@ -5,3 +5,30 @@ sol!(
     IDescribedByMetaV1,
     "../../out/IDescribedByMetaV1.sol/IDescribedByMetaV1.json"
 );
+
+sol! {
+    /// IERC165 supportsInterface fn
+    function supportsInterface(bytes4 interfaceId) public view virtual override returns (bool);
+}
+
+/// get interface id of IDescribedByMetaV1
+pub fn i_described_by_meta_v1_interface_id() -> [u8; 4] {
+    let selectors = IDescribedByMetaV1::IDescribedByMetaV1Calls::SELECTORS;
+    let mut result = u32::from_be_bytes(selectors[0]);
```

Comment:
do we not have a standard function for converting a selector to a u32?

Reply:
I dont think so, at least as far as I checked, I couldnt find anything for it other than this